src/sliding_sync.rs

```diff
@@ -357,6 +357,10 @@ pub enum MatrixRequest {
     FullyReadReceipt{
         room_id: OwnedRoomId,
         event_id: OwnedEventId,
+    },
+    /// Sends user permission's checking once user entered room.
```

Comment:
This comment isn't quite what we want to write here. You should write what this type of request actually does, not how you use it in only one part of the code.
```suggestion
    /// Sends a request checking if the currently logged-in user can send a message to the given room.
    ///
    /// The response is delivered back to the main UI thread via a `TimelineUpdate::CanUserSendMessage`.
```